mindsdb_native/libs/phases/model_interface/lightwood_backend.py

```diff
@@ -552,10 +558,12 @@ def predict(self, mode='predict', ignore_columns=None, all_mixers=False):
 
             predictions = self.predictor.predict(when_data=run_df)
 
-
             if self.transaction.lmd['quick_predict']:
                 for k in predictions:
                     formated_predictions[k] = predictions[k]['predictions']
+                    if 'class_distribution' in predictions[k]:
```

Comment:
Better to check `if self.transaction.lmd['output_class_distribution']` maybe, to avoid certain silent failures.